extensions/expression-language/Expression.cpp

```diff
@@ -628,7 +632,7 @@ Value expr_toDate(const std::vector<Value> &args) {
   auto arg_0 = args[0].asString();
   std::istringstream arg_s { arg_0 };
   date::sys_time<std::chrono::milliseconds> t;
-  arg_s >> date::parse(args[1].asString(), t);
+  date::from_stream(arg_s, args[1].asString().c_str(), t);
```

Comment:
These lines should be equivalent, but the first one failed to compile on MSVC for some reason. The error message was not very helpful to me.
https://github.com/szaszm/nifi-minifi-cpp/runs/3024143793?check_suite_focus=true#step:7:16514